docs/standard/datetime/access-utc-and-local.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: How to: access the predefined UTC and local time zone objects
```

Comment:
We established as a pattern to enclose the title with quotes in these cases. See our [style guide](https://github.com/dotnet/core-docs/blob/master/styleguide/template.md#metadata)